zwave_js/DOCS.md

```diff
@@ -176,6 +181,12 @@ the introduction of S2 security inclusion in zwave-js, this option has been
 deprecated in favor of `s0_legacy_key`. If still set, the `network_key` value will be
 migrated to `s0_legacy_key` on first startup.
 
+## Custom device config files
```

Comment:
I'm not in favor of adding this feature. The risk of introducing bugs that we'll waste time to troubleshoot in support is high. We've already seen that happen for users using Z-Wave JS UI. Why do you need custom device config?

Reply:
if a user wants to add support for a device that doesn't have a config file, they would need something like this to do so. Same for if there is a mistake with the existing config file and they need to test a fix.

With that being said, definitely understand the concern about increasing support costs and introducing bugs that would be hard to find on our side.

Since device config files primarily affect:
1. manufacturer/product name
2. Manuals/reset instructions
3. Config parameters

Would those problem spaces still be a concern from a troubleshooting perspective?

Reply:
FWIW driver logs tell if a user-provided config file was loaded. That requires the log to include the startup sequence though, which is kinda difficult to capture in HA currently.

Reply:
Being able to log to file will help with that

Reply:
There is also the `isEmbedded` property of the device config which would be exposed in the device diagnostics. We could also consider adding an alert to the device info page when a user is using a user provided config (e.g. `node.is_embedded == False`)

Reply:
thinking more about this I think adding an alert for this to the device info page is a good idea in general, thoughts?

Reply:
If you aren't able to set up a development environment I don't think you should change your device config either.

Reply:
I personally think the concern is more than the actual risk, but this should be a feature that is rarely used, so I am OK to remove it 👍🏾